### PR TITLE
chore/merge main@26ea6f1

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/f7312f7f3ae7a28c8dee8b8d84d6f285ad3cecff
- [x] https://github.com/twbs/bootstrap/commit/f6cb4b64b57f751df4563fe80e0f4eb4c19fb8e4
- [x] https://github.com/twbs/bootstrap/commit/fe257823ecca31cf5e582e9b8380a0ad0204075e
- [x] https://github.com/twbs/bootstrap/commit/33992b7d5b2176d90aee9a1933219b498a9888d4
- [x] https://github.com/twbs/bootstrap/commit/8aaeb3cb6b5fe4f3eb6ed5872e172404aa8397a6
- [x] https://github.com/twbs/bootstrap/commit/36e75f63559396c1e608c9d6599a0b9ca7512d8d
- [x] https://github.com/twbs/bootstrap/commit/3984ebb4073a2a9f0b922ab196f3d85600c30c7f
- [x] https://github.com/twbs/bootstrap/commit/7ff974b4ce51a5dd5485d62264b9de9842ea2b58
- [x] https://github.com/twbs/bootstrap/commit/74be17edef89275ad71c1448b11c72a328c8eeb5
- [x] https://github.com/twbs/bootstrap/commit/1fed10dfb15aeb6c09b9c7f81ffe9214186ff48a
- [x] https://github.com/twbs/bootstrap/commit/87da34014a6479e94130f04564898292dbc7586d
- [x] https://github.com/twbs/bootstrap/commit/cfd2f3f7787ba22feb78d916956f6f73746f3ee3
- [x] https://github.com/twbs/bootstrap/commit/ece16012270a9ef7781ce9269cb151c5e5961734
- [x] https://github.com/twbs/bootstrap/commit/d059c1d5bafb46e6d4adde07059f3373f47fa1e2
- [x] https://github.com/twbs/bootstrap/commit/7a8459641f14abeaa1717dba79fdc16201a11aa8
- https://github.com/twbs/bootstrap/commit/fb21233098124c95c9fdfd54cb546739051464c7: Nothing to do
- [x] https://github.com/twbs/bootstrap/commit/fa0adc979d0c60b1b0ef00e034cfe0c86950ea53
- [x] https://github.com/twbs/bootstrap/commit/26ea6f1649c51d2a7edc796221bd5b46b289bb28

### CODE REVIEW

File: site/content/docs/5.1/components/dropdowns.md [hunk 1]
```diff
@@ -741,25 +737,30 @@ Put a form within a dropdown menu, or make it into a dropdown menu, and use [mar
...
     </div>
-  </div>
-  <button type="submit" class="btn btn-primary">Sign in</button>
-</form>
+    <div class="mb-3">
+      <label for="exampleDropdownFormPassword2" class="form-label">Password</label>
+      <input type="password" class="form-control" id="exampleDropdownFormPassword2" placeholder="Password">
+    </div>
+    <div class="mb-3">
+      <div class="form-check">
```
Comment (julien-deramond) [inline site/content/docs/5.1/components/dropdowns.md:754]: [Bootstrap] A click in "Remember" closes the dropdown and shouldn't. There is an existing issue about this new dropdown but its content is not exactly the same.

File: scss/_tooltip.scss [hunk 2]
```diff
@@ -18,10 +18,10 @@
   --#{$prefix}tooltip-arrow-height: #{$tooltip-arrow-height};
   // scss-docs-end tooltip-css-vars
 
-  position: absolute;
   z-index: var(--#{$prefix}tooltip-zindex);
   display: block;
   margin: var(--#{$prefix}tooltip-margin);
+  @include deprecate("`$tooltip-margin`", "v5", "v5.x", );
```
Comment (julien-deramond) [inline scss/_tooltip.scss:24]: [Bootstrap] Shouldn't we remove the corresponding CSS var as well?

File: site/content/docs/5.1/migration.md [hunk 3]
```diff
@@ -124,6 +124,8 @@ Your custom Boosted CSS builds should now look something like this with a separa
 
 - **Added new snippet examples based to show how to customize our components. —** Pull ready to use customized components and other common design patterns with our new [Snippets examples]({{< docsref "/examples#snippets" >}}). Includes [footers]({{< docsref "/examples/footers/" >}}), [dropdowns]({{< docsref "/examples/dropdowns/" >}}), [list groups]({{< docsref "/examples/list-groups/" >}}), and [modals]({{< docsref "/examples/modals/" >}}).
 
+- **Removed unused positioning styles from popovers and tooltips** as these are handled solely by Popper.js. `$tooltip-margin` has been deprecated and set to `null` in the process.
```
Comment (julien-deramond) [inline site/content/docs/5.1/migration.md:127]: [Bootstrap] This is not in the appropriate section (already fixed in https://github.com/twbs/bootstrap/commit/3362c8cf13cc3610a6e0ac1efc673f7f8c74ac08)

File: scss/_popover.scss [hunk 4]
```diff
@@ -24,9 +25,6 @@
   --#{$prefix}popover-arrow-border: var(--#{$prefix}popover-border-color);
   // scss-docs-end popover-css-vars
 
-  position: absolute;
-  top: 0;
-  left: 0 #{"/* rtl:ignore */"};
```
Comment (julien-deramond) [inline scss/_popover.scss:29]: [Bootstrap / Boosted] Check this use case

File: site/content/docs/5.1/components/dropdowns.md [hunk 5]
```diff
@@ -457,15 +455,13 @@ Trigger dropdown menus at the left of the elements by adding `.dropstart` to the
...
-<div class="btn-group" role="group">
-  <div class="btn-group dropstart" role="group">
-    <button type="button" class="btn btn-secondary dropdown-toggle dropdown-toggle-split" data-bs-toggle="dropdown" aria-expanded="false">
-      <span class="visually-hidden">Toggle Dropstart</span>
-    </button>
-    <ul class="dropdown-menu">
-      <!-- Dropdown menu links -->
-    </ul>
-  </div>
+<div class="btn-group dropstart" role="group">
```
Comment (julien-deramond) [inline site/content/docs/5.1/components/dropdowns.md:458]: [Bootstrap] @louismaximepiton In https://raw.githubusercontent.com/twbs/bootstrap/main/site/content/docs/5.1/components/dropdowns.md `role="group"` has been added only for the dropstart.

File: site/content/docs/5.1/components/scrollspy.md [hunk 6]
```diff
@@ -134,20 +128,34 @@ Scrollspy also works with nested `.nav`s. If a nested `.nav` is `.active`, its p
...
-        <h4 id="item-3">Item 3</h4>
-        <p>This is some placeholder content for the scrollspy page. Note that as you scroll down the page, the appropriate navigation link is highlighted. It's repeated throughout the component example. We keep adding some more example copy here to emphasize the scrolling and highlighting.</p>
-        <h5 id="item-3-1">Item 3-1</h5>
-        <p>This is some placeholder content for the scrollspy page. Note that as you scroll down the page, the appropriate navigation link is highlighted. It's repeated throughout the component example. We keep adding some more example copy here to emphasize the scrolling and highlighting.</p>
-        <h5 id="item-3-2">Item 3-2</h5>
-        <p>This is some placeholder content for the scrollspy page. Note that as you scroll down the page, the appropriate navigation link is highlighted. It's repeated throughout the component example. We keep adding some more example copy here to emphasize the scrolling and highlighting.</p>
+        <div id="item-1">
+            <h4>Item 1</h4>
```
Comment (julien-deramond) [inline site/content/docs/5.1/components/scrollspy.md:132]: [Bootstrap] Too much tabs + \```html``` not updated

File: site/assets/scss/_component-examples.scss [hunk 7]
```diff
@@ -271,18 +271,31 @@
...
+    margin: .5rem 0;
+
+    // Boosted mod: no &:focus rule
+  }
+
+  .active {
+    // Boosted mod: no background-color
+    font-weight: $font-weight-bold; // Boosted mod
+  }
+
```
Comment (julien-deramond) [inline site/assets/scss/_component-examples.scss:296]: [Bootstrap] Extra break line

File: site/content/docs/5.1/examples/cheatsheet/index.html [hunk 8]
```diff
@@ -1156,13 +1156,13 @@ <h3>Navs</h3>
           </nav>
           <div class="tab-content" id="nav-tabContent">
             <div class="tab-pane fade show active" id="nav-home" role="tabpanel" aria-labelledby="nav-home-tab">
-              <p><strong>This is some placeholder content the Home tab's associated content.</strong> Clicking another tab will toggle the visibility of this one for the next. The tab JavaScript swaps classes to control the content visibility and styling. You can use it with tabs, pills, and any other <code>.nav</code>-powered navigation.</p>
```
Comment (julien-deramond) [inline site/content/docs/5.1/examples/cheatsheet/index.html:1159]: [Bootstrap] Create a PR to make the same modification. Forgot to do it after the modification in nav tabs.

File: site/content/docs/5.1/components/scrollspy.md (hunk 6 above)
Comment (julien-deramond) [reply site/content/docs/5.1/components/scrollspy.md:132]: See https://github.com/twbs/bootstrap/pull/36195

File: site/assets/scss/_component-examples.scss (hunk 7 above)
Comment (julien-deramond) [reply site/assets/scss/_component-examples.scss:296]: See https://github.com/twbs/bootstrap/pull/36195

File: site/content/docs/5.1/examples/cheatsheet/index.html (hunk 8 above)
Comment (julien-deramond) [reply site/content/docs/5.1/examples/cheatsheet/index.html:1159]: See https://github.com/twbs/bootstrap/pull/36196

File: site/content/docs/5.1/components/dropdowns.md (hunk 1 above)
Comment (louismaximepiton) [reply site/content/docs/5.1/components/dropdowns.md:754]: Maybe an `data-bs-auto-close="outside"` would do the trick in here ?

File: site/content/docs/5.1/components/scrollspy.md [hunk 9]
```diff
@@ -244,18 +252,63 @@ Scrollspy also works with `.list-group`s. Scroll the area next to the list group
...
+        <a href="#simple-list-item-1">Item 1</a>
+        <a href="#simple-list-item-2">Item 2</a>
+        <a href="#simple-list-item-3">Item 3</a>
+        <a href="#simple-list-item-4">Item 4</a>
+        <a href="#simple-list-item-5">Item 5</a>
+      </div>
+    </div>
+    <div class="col-8">
+      <div data-bs-spy="scroll" data-bs-target="#simple-list-example" data-bs-offset="0" data-bs-smooth-scroll="true" class="scrollspy-example" tabindex="0">
+        <h4 id="simple-list-item-1">Item 1</h4>
```
Comment (louismaximepiton) [inline site/content/docs/5.1/components/scrollspy.md:272]: Do we need to add some changes everywhere with a surrounding `<div>` ?

**Review by louismaximepiton — APPROVED** [review]
LGTM since all breaking changes appear on Bootstrap's side. Maybe one or two comment on our side ?

Reminder for me : 
- sw.js
- Scrollspies seems to be broken on Bootstrap side

File: site/assets/js/tac.js [hunk 10]
```diff
@@ -67,21 +65,16 @@
...
+      const toggle = document
         .createRange()
         .createContextualFragment(
-          '<div class="form-check form-switch my-2 my-lg-0"><input class="form-check-input ms-0" type="checkbox" id="googletagmanagerAllowed" aria-describedby="tacCLgoogletagmanager" onchange="' +
-            choiceEvent +
-            '"' +
-            ((document.cookie.match(/^(?:.*;)?\s*cookie-consent\s*=\s*([^;]+)(?:.*)?$/) || [null])[1] === '!googletagmanager=true' ? 'checked' : '') +
-            '><label class="form-check-label visually-hidden" for="googletagmanagerAllowed">Google Tag Manager</label>' +
-            '<input id="googletagmanagerDenied" class="d-none"></div>'
+          `<div class="form-check form-switch my-2 my-lg-0"><input class="form-check-input ms-0" type="checkbox" id="googletagmanagerAllowed" aria-describedby="tacCLgoogletagmanager" onchange="${choiceEvent}"${((document.cookie.match(/^(?:.*;)?\s*cookie-consent\s*=\s*([^;]+)(?:.*)?$/) || [null])[1] === '!googletagmanager=true' ? 'checked' : '')}><label class="form-check-label visually-hidden" for="googletagmanagerAllowed">Google Tag Manager</label><input id="googletagmanagerDenied" class="d-none"></div>`
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/assets/js/tac.js:77]: Agree but I haven't found a way while I was doing it to keep the multi-line approach with the back ticks. Any idea?

File: js/src/scrollspy.js [hunk 11]
```diff
@@ -21,34 +20,34 @@ const EVENT_KEY = `.${DATA_KEY}`
...
-  target: ''
+  offset: null, // TODO: v6 @deprecated, keep it for backwards compatibility reasons
+  rootMargin: '0px 0px -25%',
+  smoothScroll: false,
+  target: null
 }
 
 const DefaultType = {
-  offset: 'number',
-  method: 'string',
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply js/src/scrollspy.js:50]: Check how it is done for other components and if it should be mentioned (if not already the case) in the migration guide.

File: site/content/docs/5.1/components/scrollspy.md (hunk 9 above)
Comment (julien-deramond) [reply site/content/docs/5.1/components/scrollspy.md:272]: I haven't looked at the new code for the scrollspy but I've found it weird; a different HTML structure for this use case.

**Pull request merged**